src/main/java/be/seeseemelk/mockbukkit/block/BlockMock.java

```diff
@@ -344,14 +348,12 @@ public int getBlockPower()
 	@Override
 	public boolean isEmpty()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.getTypeId() == 0;
```

Comment:
```suggestion
		return this.getType().equals(Material.AIR);
```
This smells like a magic value a little